src/cascadia/TerminalSettingsEditor/ColorSchemes.cpp

```diff
@@ -53,9 +56,58 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
 
     ColorSchemes::ColorSchemes() :
         _ColorSchemeList{ single_threaded_observable_vector<Model::ColorScheme>() },
-        _CurrentColorTable{ single_threaded_observable_vector<Editor::ColorTableEntry>() }
+        _CurrentNonBrightColorTable{ single_threaded_observable_vector<Editor::ColorTableEntry>() },
+        _CurrentBrightColorTable{ single_threaded_observable_vector<Editor::ColorTableEntry>() }
     {
         InitializeComponent();
+
+        const auto foregroundButton{ ForegroundButton() };
```

Comment:
It technically compiles down to "free" to just call `ForegroundButton()` repeatedly

Reply:
fwiw: i _hate_ this :smile: